fiftyone/utils/fcclip.py

```
@@ -0,0 +1,139 @@
+        d = self.init(d)
+        super().__init__(d)
+        self.name_or_path = self.parse_string(d, "name_or_path")
+        self.score_threshold = self.parse_number(
+            d, "score_threshold", default=0.8
+        )
+        self.class_names = self.parse_array(d, "class_names", default=None)
+        self.validate_config()
+
+    def validate_config(self):
+        if not 0 <= self.score_threshold <= 1:
+            raise ValueError("score_threshold must be between 0 and 1")
```

Ruff (0.15.18)

[warning] Avoid specifying long messages outside the exception class

(TRY003)